Add Breakin

fabaff opened the pull request:
Breakin is Advanced Clustering’s stress-test and hardware diagnostics tool.
http://www.advancedclustering.com/products/software/breakin/

antonym commented on src/utils.ipxe:
```diff
@@ -41,6 +42,12 @@ set util_path download.avg.com/filedir/inst/avg_arl_cdi_all_120_${avg_version}.i
+:breakin
+set breakin_version 4.26.1-53
+set util_path www.advancedclustering.com/wp-content/uploads/2017/02/bootimage-${breakin_version}.iso
+set util_file bootimage-${breaking_version}.iso
```
should be ${breakin_version}

fabaff committed:
Add Breakin

Breakin is Advanced Clustering’s stress-test and hardware diagnostics tool.
http://www.advancedclustering.com/products/software/breakin/
